docs/src/pages/components/data-grid/columns/RenderExpandCellGrid.js

```diff
@@ -118,7 +118,7 @@ function renderCellExpand(params) {
   return (
     <GridCellExpand
       value={params.value ? params.value.toString() : ''}
-      width={params.colDef.width}
+      width={params.colDef.computedWidth}
```

Comment:
This PR could be considered breaking by some, e.g. say the user copied this demo in the past and uses flex: 1. https://codesandbox.io/s/material-demo-forked-zpk7c?file=/demo.js. I would have personally not used "core" as the primary label, but it's a detail. We will see if people complain.

Reply:
You're right
I guess it depends if you want to be exhaustive in your breaking change or if you want to focus on the main ones.
We can update the label if you prefer ?

Reply:
> I guess it depends if you want to be exhaustive in your breaking change or if you want to focus on the main ones.

It's a long time dilemma discussion @eps1lon and I have had. I personally think that there is a grey scale, bug fixes change behaviors, wrong behaviors developers might rely on.

When developers raise a change of behavior in an issue that wasn't documented in the changelog or versioning, it's usually a good sign that we haven't been strict enough

At the end of the day, I think that it's about proving the best update experience as possible for developers. Over communication might get us into https://overreacted.io/npm-audit-broken-by-design/, but user communication might even be worse.

Reply:
npm-audit is problematic because we can't communicate as well as we can with SemVer. This is a breaking change. A breaking change is not an "opinion" or grey area. It is well defined.